config.py.example

```diff
@@ -64,41 +64,37 @@ LINK_SHORTENER_SERVICE = 'dagd'
 LINK_SHORTENER_PARAMS =  {}
 
 # For datetime nonsense 
-TIMEZONE='America/Toronto'
+TIMEZONE='America/Los_Angeles'
 
 # Used as the link field in the RSS feed
-WEBSITE='http://watcamp.com'
+WEBSITE='https://blm-cal.tech-bloc-sea.dev/'
 
 # Used as the location of this feed
-FEED_LINK="{}/watcamp.rss".format(WEBSITE)
+FEED_LINK="{}/seattleblmcalendar.rss".format(WEBSITE)
```

Comment:
So I take it we would need to serve the file at this URL?

Reply:
Yeah we'd want to serve it from `https://blm-cal.tech-bloc-sea.dev/seattleblmcalendar.rss` I guess. We can name it anything we want, `feed.rss` even 🤷 

Which reminds me I need to fix the backslack in the website constant.